docs/nodejs_example.md

```diff
@@ -1,9 +1,17 @@
 ## Setup
 
 * Install [Node.js](https://nodejs.org) (6 or higher)
-* Install the following node modules with [yarn](https://yarnpkg.com/en/) or [npm](https://www.npmjs.com/)
-  * chai@latest
-  * cucumber@latest
+* Install the last version of [Chai](https://www.chaijs.com/) and Cucumber modules with [yarn](https://yarnpkg.com/en/) 
+```
+yarn add -D chai@latest cucumber@latest
+```
+
+* or with [npm](https://www.npmjs.com/)
+
+```
+npm i -D chai@latest cucumber@latest
+```
```

Comment:
Thoughts on leaving this as one bullet and the two snippets are together like:
```
yarn add -D chai@latest cucumber@latest
# or 
npm i -D chai@latest cucumber@latest
```
Also please indent the code snippet two spaces so it goes under the bullet

Reply:
Done, @charlierudolph, pls, let me know if there is something else to improve in this file.